VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -111,10 +112,10 @@ export class AppComponent implements OnInit, OnDestroy {
                             service. This method is used to identify the user in Dynatrace
                             monitoring by passing the user's preferred username in lowercase as a
                             parameter.*/
-                            // TODO: uncomment the below line to add this identification
-                            // To be implemented behind the cookie banner VIH-5622
-
-                            // this.dynatraceService.addUserIdentifyScript(userData?.preferred_username?.toLowerCase());
+                            const cookieConsent = localStorage.getItem(cookies.cookieConsentKey);
+                            if (cookieConsent === 'accepted') {
```

Comment:
If already authenticated, this block of code does not execute and so the username is never set.

Reply:
done

Reply:
done